src/helpers.js

```diff
@@ -0,0 +1,55 @@
+const sha1 = require('sha1');
+
+export const checksum = str => parseInt(sha1(str).substr(0, 8), 16);
+
+export const getLastItem = obj => {
+  const keys = Object.keys(obj);
+  const lastKey = keys[keys.length - 1];
+
+  const item = obj[lastKey];
+  delete obj[lastKey];
+
+  return item;
+};
+
+export const getSql = () => {
+  let sql;
+  if (process.env.APP_ENV === 'browser') {
+    require('script!sql.js');
+    sql = window.SQL;
+  } else {
+    sql = require('sql.js');
+  }
+  return sql;
+};
+
+export const rand = () => Math.random() * 100000000 | 0;
+
+export const getDb = () => new (getSql().Database)();
+
+export const getZip = (...args) => {
```

Comment:
I still think that it be good if we able to provide dependencies ( db / zip ) from outside

It decrease coupling and make testing easier.

Yep. For this small project it's not critical, but it's pretty way